index.html

```diff
@@ -7,13 +7,27 @@
   </head>
   <body>
     <header>
-
+      <h1>INTENTION TIMER</h1>
     </header>
     <main>
-      <section>
-        
+      <section class="new-activity">
+        <h2>New Activity</h2>
+        <form>
+          <p class="select-category">Select a category:</p>
+          <button class="category-button study-button"><img scr="./assets/study.svg">Study</button>
```

Comment:
This <img> does not work. Still figuring that out. Apart from that, this is looking really good.